source/onboard/migrate-from-slack.rst

```diff
@@ -0,0 +1,191 @@
+5. Select **OAuth & Permissions** on the left-hand side of the screen. Then scroll down to **Scopes**.
+6. Under **Bot Token Scopes** type in and select the following scopes:
+  - ``users:read``
+  - ``users:read.email``
+
+7. Scroll up and select **Install to Workspace**.
+8. Grant the app permissions when prompted.
+9. Copy the **Bot User OAuth Token** and save it somewhere convenient.
+
+We'll now use **Bot User OAuth Token** with the ``slack-advanced-exporter`` tool to download emails and attachments. Download the latest release of ``slack-advanced-exporter`` for your OS and architecture `here <https://github.com/grundleborg/slack-advanced-exporter/releases/>`__ and extract the executable from the download.
+
+Once you have the program downloaded locally, run the commands below to fetch the emails, and then fetch file attachments. Replace ``<SLACK TOKEN>`` with the Slack token you generated earlier and ``<SLACK EXPORT FILE>`` with the `path <https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/>`__ to your export file.
+
+.. code:: bash
+
+  ./slack-advanced-exporter --input-archive <SLACK EXPORT FILE> --output-archive export-with-emails.zip fetch-emails --api-token <SLACK TOKEN>
+  ./slack-advanced-exporter --input-archive export-with-emails.zip --output-archive export-with-emails-and-attachments.zip fetch-attachments --api-token <SLACK TOKEN>
+
+.. note::
+
+  - The first command collects all of the user emails and creates the file ``export-with-emails.zip``. The second command fetches attachments and creates the file ``export-with-emails-and-attachments.zip``, which we will use going forward.
+  - The second command can take a long time if you have a large number of file uploads. If it's interrupted, delete the file generated (if any), and start again.
+
+The file ``export-with-emails-and-attachments.zip`` now contains all the information necessary to be imported into Mattermost.
+
+From the slack advanced exporter repo:
+Due to archive/zip limitations, these actions cannot modify archive in place. It's preferable to fetch e-mails first to avoid copying large attachments around.
+
+.. important::
+
+  Avoid unzipping and rezipping the Slack export. Doing so can modify the directory structure of the archive which could cause issues with the import process.
+
+4. Convert Slack import to Mattermost's bulk export format
+----------------------------------------------------------
+
+Now that you have a Slack export file with emails and attachments, let's convert this information into Mattermost's bulk import format using the import preparation tool ``mmetl``. Download the latest release of ``mmetl`` for your `OS and architecture <https://github.com/mattermost/mmetl/releases/>`__. Run ``mmetl help`` to learn more about using the tool.
+
+Next, run the following command to create a Mattermost bulk import file. Replace ``<TEAM NAME>`` with the name of your team in Mattermost:
```

Comment:
I had to use a team name without spaces. Tried `slack import` at first but it didn't work:

```
Data: map[error:Error during job execution. — BulkImport: Error importing channel. Team with name "slack import" could not be found., resource: Team id: name=slack import import_file:8jee3tjr1jd3xqi3mpbooe976r_mattermost-bulk-import.zip line_number:3]
```

After I changed it to `slack-import` it worked perfectly.